DAFF-4 sub-level content edits (#1771)
agriculturegovau/agds-next: packages/react/src/app-layout/docs/overview.mdx

### Sub-level items

> When choosing if sub-level items should be always visible or only visible when active you should consider:

Reviewer: ```suggestion
When choosing if sub-level items should be always visible or only visible when active you should consider:

```

Reviewer: Our styling linter is getting tripped because it wants an empty line before the start of a list. Same with the one below.

- How many sub-level items will each top-level item have?
- Do users need to see sub-level items to understand the structure of your site?
- Do users need to jump to sub-level items quickly to complete frequent tasks?
- Will visible sub-level items cause excessive scrolling or tab stops for keyboard users?

#### Visible when active

> Sub-level items can be revealed when the parent item is active. This option is best when the number of visible items in list will increase cognitive load or cause excessive scrolling or tab stops for keyboard users. 

Reviewer: ```suggestion
Sub-level items can be revealed when the parent item is active. This approach is best when the number of visible items in the list will increase cognitive load, for example by causing excessive scrolling and tab stops for keyboard users. 
```

`option` is a common code term for list items (we even have it in our source code), so I've suggested `approach` to avoid confusion. Some other minor suggestions, too.

> Visible when active is the default behaviour. Setting 'subLevelVisible="whenActive"' or omitting this prop will result in sub-level links that are only visible when active.

Reviewer: ```suggestion
Visible when active is the default behaviour. Setting `subLevelVisible="whenActive"` or omitting this prop will result in sub-level links that are only visible when active.
```

#### Always visible

> Making sub-level items always visible will help users if:

Reviewer: ```suggestion
Making sub-level items always visible will help users if:

```

- you have a small number of items and it’s easy for users to see all items at once without excessive scroll or tab stops,
- it will help users understand the structure of your site,
- you know users need to jump to sub-levels to quickly to complete frequent tasks.

> Setting 'subLevelVisible="always"' will result in sub-links that are always visible.

Reviewer: ```suggestion
Setting `subLevelVisible="always"` will result in sub-links that are always visible.
```
